range:
  min: 1.  # Min range (m)
  max: 300.0  # Max range (m)
  noise: 0.005  # Standard deviation of gaussian noise (m)
angle:
  min: -3.14  # The min scan angle (radians)
  max: 3.14  # The max scan angle (inclusive, radians)
  increment: 0.05  # The angle increment to scan at (radians)
map:
  topic: "/map"  # The map topic to subscribe to
  image_frame: "/map_image"  # An intemediate TF frame generated by laser_scan_sim
  materials_topic: "/map_materials"  # The materials list
  segments_topic: "/map_segments"  # The wall segment list
  circles_topic: "/map_circles"  # The circles list
laser:
  hz: 1  # laser scan frequency
  topic: "/scan"  # The LaserScan topic to publish
  frame: "/initialpose"  # The tf frame that the laser is based in
